pallets/gas/src/tests.rs

```diff
@@ -46,6 +46,41 @@ fn simple_value_tree() {
     });
 }
 
+#[test]
```

Comment:
please, provide also test, that if we cut N from node A with initial balance M to reserved node B, than value of B is N, and value of A is M - N

Reply:
https://github.com/gear-tech/gear/blob/fa9763823e7bb3f1f15f74bb252c2a30525f010f/pallets/gas/src/tests.rs#L50